modules/nested-accordion/assets/scss/frontend.scss

```diff
@@ -63,32 +88,69 @@
 			}
 
 			&[open] {
-
 			  	.e-n-accordion-item-title {
```

Comment:
empty line above

Reply:
@Barcov Meaning this empty line should be here (it should not be removed).